src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,52 @@
+.calendar {
+  display: flex;
+  flex-shrink: 0;
+  flex-wrap: wrap;
+  gap: $gap-between-columns;
+  padding: $calendar-padding;
+  width: ($column-size * $columns-in-row) +
+    ($gap-between-columns * $gaps-in-row) +
+    $calendar-padding;
+
+  @for $days from 28 through 31 {
+    &--month-length--#{$days} {
+      :nth-child(n + #{$days+1}) {
+        visibility: hidden;
```

Comment:
Better use `display: none` instead of `visibility: hidden` to completely remove the day from the page